pilot/test/integration/testdata/mixer.yaml.tmpl

```diff
@@ -33,7 +33,7 @@ spec:
       serviceAccountName: istio-mixer-service-account
       containers:
       - name: mixer
-        image: {{.Hub}}/mixer:{{.Tag}}
+        image: gcr.io/istio-testing/mixer:572c3ba624ef1c8e4f5d834fc7770f1fba72124a
```

Comment:
Is there a reason to fix this to a particular version?

Reply:
No. Thanks for noticing it. I changed it to run e2e test locally only.